src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs

```diff
@@ -22,31 +23,138 @@ public GetInvitationByIdQueryHandler(IReadOnlyContext context, IFusionMeetingCli
+                }
+
+                foreach (var personInFunctionalRole in participant.FunctionalRole.Persons)
+                {
+                    var participantType = GetParticipantTypeByEmail(meeting, personInFunctionalRole.Email);
+                    personInFunctionalRole.Required = participantType.Equals(ParticipantType.Required);
+
+                    var functionalRolePersonResponse = GetOutlookResponseByEmailAsync(meeting, personInFunctionalRole.Email);
+                    personInFunctionalRole.Response = functionalRolePersonResponse;
+                }
+
+                if (participant.ExternalEmail != null)
+                {
+                   var externalEmailResponse = GetOutlookResponseByEmailAsync(meeting, participant.ExternalEmail?.ExternalEmail);
+                    participant.ExternalEmail.Response = externalEmailResponse;
+                }
+
+                if (participant.FunctionalRole?.Email != null)
+                {
+                    var functionalRoleResponse = GetOutlookResponseByEmailAsync(meeting, participant.FunctionalRole.Email);
+                    participant.FunctionalRole.Response = functionalRoleResponse;
+                }
+            }
         }
+
+        private static OutlookResponse? GetOutlookResponseByEmailAsync(GeneralMeeting meeting, string email)
+            => meeting.Participants.SingleOrDefault(p 
+            => string.Equals(p.Person.Mail, email, StringComparison.CurrentCultureIgnoreCase))
+            ?.OutlookResponse;
+
+        private static ParticipantType? GetParticipantTypeByEmail(GeneralMeeting meeting, string email) 
+            => meeting.Participants.SingleOrDefault(p => p.Person.Mail == email)?.Type;
+
+        private static IEnumerable<CommPkgScopeDto> ConvertToCommPkgDto(IEnumerable<CommPkg> commPkgs)
+            => commPkgs.Select(commPkg => new CommPkgScopeDto(commPkg.CommPkgNo, commPkg.Description, commPkg.Status));
+
+        private static IEnumerable<McPkgScopeDto> ConvertToMcPkgDto(IEnumerable<McPkg> mcPkgs) 
+            => mcPkgs.Select(mcPkg => new McPkgScopeDto(mcPkg.McPkgNo, mcPkg.Description, mcPkg.CommPkgNo));
+
+        private static IEnumerable<ParticipantDto> ConvertToParticipantDto(IReadOnlyCollection<Participant> participants)
+        {
+            var participantDtos = new List<ParticipantDto>();
+
+            foreach (var participant in participants)
+            {
+                if (participant.Type == IpoParticipantType.FunctionalRole)
+                {
+                    var personsInFunctionalRole = participants
+                        .Where(p => p.FunctionalRoleCode == participant.FunctionalRoleCode 
+                         && p.Type == IpoParticipantType.Person);
+
+                    participantDtos.Add(new ParticipantDto(participant.Organization, participant.SortKey, null,
+                        null, ConvertToFunctionalRoleDto(participant, personsInFunctionalRole), participant.RowVersion.ConvertToString()));
```

Comment:
RowVersion not on participantDto, but on ExternalEmailDto, FunctionalRoleDto, and PersonDto